single_tech_samples/streamanalytics/e2e/e2e.ts

```diff
@@ -18,12 +19,12 @@ describe('Send to IoT Hub', () => {
     });
 
     before(async () => {
-        iotClient = Client.fromConnectionString(process.env.DEVICE_CONNECTION_STRING, Mqtt);
+        iotClient = Client.fromConnectionString(process.env.DEVICE_CONNECTION_STRING as string, Mqtt);
         await iotClient.open();
     });
 
     before(() => {
-        containerClient = new ContainerClient(process.env.AZURE_STORAGE_CONNECTION_STRING, EVENT_SINK_CONTAINER);
+        containerClient = new ContainerClient(process.env.AZURE_STORAGE_CONNECTION_STRING as string, EVENT_SINK_CONTAINER);
```

Comment:
is this casting to string necessary? for example, if this is not a string, maybe the right thing to do is to throw an exception, rather than casting the incorrect thing to a string

Reply:
Discussed offline. closing